jekyll/_cci2/server-3-install.adoc

```diff
@@ -143,17 +143,84 @@ The domain you selected for your CircleCI installation is the Homepage URL and *
+. MongoDB authentication source database: The database that holds the account information, usually `admin`.
+. MongoDB authentication mechanism: The authentication mechanism to use, usually `SCRAM-SHA-1`.
+. Additional connection options: Any other connection options you would like to use. This needs to be formatted as a query string (key=value pairs, separated by &, special characters need to be URL encoded). See the https://docs.mongodb.com/v3.6/reference/connection-string/[MongoDB docs] for available options.
+
+### Vault
+*Internal* will deploy the default Vault instance inside the CircleCI K8s namespace.  The application will be automatically configured.
+*External* will not install the default Vault instance with the CircleCI application.  Select this option if using an existing Vault instance.  You need to configure the following settings:
+
+. URL: ex. `http://vault:8200`
+. Transit Path: The path of the transit secrets engine.  The default is `transit`. See the https://www.vaultproject.io/docs/secrets/transit#setup[Vault documentation] for more details.
+. Token: The Vault token to be used by CircleCI.  The following example shows how to create a token with the recommended policy
+
+Create the policy:
+[source,sh]
+----
+vault policy write circleci -<<EOF
+path "mytransit/export/*" {
+  capabilities = ["deny"]
+}
+path "mytransit/encrypt/*" {
+  capabilities = ["update"]
+}
+path "mytransit/decrypt/*" {
+  capabilities = ["update"]
+}
+path "/auth/token/lookup-self" {
+    capabilities = ["read", "list"]
+}
+EOF
+
+vault token create -policy=circleci
+----
+
+Create a token using the policy:
+[source,sh]
+----
+vault token create -policy=circleci -period=30m
+----
+
 ### Object Storage
-Server 3.0 hosts build artifacts, test results, and other state in object storage. Please choose the one that best suits your needs.
-A *Storage Bucket Name* is required, in addition to the following fields, depending on if you are using AWS or GCP. Ensure
-that the bucket name you provide exists in your chosen object storage provider before proceeeding.
 
-#### AWS S3
-. *AWS S3* (required): Region your S3 bucket is in.
-. *AWS IAM Access Key ID* (required): https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[AWS Access Key ID] for S3 bucket access.
-. *AWS IAM Secret Key* (required): https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[IAM Secret Key] for S3 bucket access.
+Server 3.x hosts build artifacts, test results, and other state in object storage. We support
+
+. Any https://docs.aws.amazon.com/s3/index.html?nc2=h_ql_doc_s3[S3]-compatible storage
+. https://cloud.google.com/storage/[Google Cloud Storage]
+
+While any S3-compatible object storage may work, we test and support https://aws.amazon.com/s3/[AWS S3] and https://min.io[Minio]. For object storage
+providers that do not support the https://docs.aws.amazon.com/AmazonS3/latest/API/Type_API_Reference.html[S3 API], such as
```

Comment:
@nsmith5 Hey, here we say we support any S3 storage but then say we suport AWS S3 and Minio, but others may work - so I guess not officially supported. My question is would it be better for the bullet list  to say AWS S3, Minio and GCS are supported?

Reply:
Right, that is definitely contradictory. I think the list saying AWS S3, Minio and GCS are supported makes sense like you say